internal/pkg/client/oras/resolver.go

```diff
@@ -0,0 +1,133 @@
+package oras
```

Comment:
Please include the standard header on this file, and also include a comment to say what the file is for, to add progress bars.

Reply:
thanks! added
https://github.com/apptainer/apptainer/pull/1587/commits/aab871d7c4c217c169908a6e6afddc9f49654c3f